crates/icrate/src/WebKit/mod.rs

```diff
@@ -0,0 +1,7 @@
+#[path = "../generated/WebKit/mod.rs"]
+mod generated;
+
+pub use self::generated::*;
+
+#[cfg_attr(feature = "apple", link(name = "WebKit", kind = "framework"))]
```

Comment:
Nit: Always link to `WebKit`, similar to e.g. `AuthenticationServices`
```suggestion
#[link(name = "WebKit", kind = "framework")]
```